build-bot-status.py

```diff
@@ -0,0 +1,95 @@
+# -*- coding: utf-8 -*-
+import sys
+import time
+import threading
+
+import requests
+
+from light_controller import LightController
+
+
+class BuildBotSession:
+    failColour = "red"
+    busyColour = "orange"
+    goodColour = "green"
+    exceptionColour = "purple"
+
+    def __init__(self, builderURL, slaveName):
```

Comment:
Documentation please. what is slaveName?

Reply:
LightController and BuildBotService should be pass in as a parameter. so that we can easily test this Controller system